<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyForm</class>
 <widget class="QDialog" name="MyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exam4_Part2 by Michael Gain</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee</family>
    </font>
   </property>
   <property name="gridVisible">
    <bool>true</bool>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::SingleLetterDayNames</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Julian Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelJulianDate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConvert">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee</family>
    </font>
   </property>
   <property name="text">
    <string>Convert To Julian</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee</family>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
